forge/test/operators/utils/compat.py

```diff
@@ -0,0 +1,80 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# Compatibility methods and datatypes for Forge
+
+import forge
+import torch
+
+from loguru import logger
+from typing import Optional, List
+
+from forge import ForgeModule, Module, VerifyConfig
+from forge.op_repo import TensorShape
+from forge.op.eval.common import compare_with_golden
+
+
+# TODO - Remove this class once TestDevice is available in Forge
+class TestDevice:
+
+    pass
+
+
+def get_dtype_for_df(dev_data_format: forge.DataFormat = None) -> torch.dtype:
+
+    dtype: torch.dtype
+
+    if dev_data_format is None:
+        dtype = None
+    else:
+        dtype: forge.dtype = torch.float32
+
+        # if dev_data_format in (
+        #     forge.DataFormat.Float16,
+        #     forge.DataFormat.Float16_b,
+        # ):
+        #     dtype = torch.float16
+        # elif dev_data_format in (
+        #     forge.DataFormat.Int8,
+        # ):
+        #     dtype = torch.float16
+
+    dtype = None
+
+    return dtype
+
+
+# Compatibility method for verifying models
+def verify_module(model: Module, input_shapes: List[TensorShape], check_pcc: Optional[bool] = True, pcc: Optional[float] = None, dev_data_format: forge.DataFormat = None):
+
+    logger.debug(f"Verifying model class: {model.__class__.__name__}({model.__class__.__base__.__module__}.{model.__class__.__base__.__name__}) input_shapes: {input_shapes}")
+
+    # TODO configure manual seed
+    generator = torch.Generator().manual_seed(42)
+
+    dtype = get_dtype_for_df(dev_data_format)
+
+    # if dtype is not None:
+    #     torch.set_default_dtype(dtype)
+
+    # forge.config.set_configuration_options(default_df_override=dev_data_format)
```

Comment:
@nvukobratTT @dgolubovicTT Is it needed to specify default_df_override for single op tests?

Reply:
This is answered. It's not.